src/components/molecules/ZopaFooter/styles/Footer.tsx

```diff
@@ -0,0 +1,6 @@
+import styled from 'styled-components';
+import { spacing } from '../../../../constants/spacing';
+
+export const Footer = styled.footer`
+  margin-bottom: ${spacing[9]};
```

Comment:
you can probably remove it and just add an atomic class directly

Reply:
mmm... our component starts with this:

```
<Footer {...rest> />
```

I can't add a `className` prop, because we will overwrite external classes.

Example:
- Somebody needs an extra class <Footer className="lol" />